docs/05-dev-tools/node-rpc/rpc-api.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 5
+title: RPC API
+sidebar_label: RPC API
+tags: [rsk, rootstock, resources, tutorials,  setup, dApps, protocol, RPC, API, ]
+description: "Remote Procedure Call (RPC) is a protocol that allows a program to execute procedures (functions) on a remote server as if they were local calls."
+---  
+ 
+Remote Procedure Call (RPC) is a protocol that allows a program to execute procedures (functions) on a remote server as if they were local calls. This eliminates the need for developers to handle low-level network communication.  
+
+### How Our API Works  
+Our API uses the JSON-RPC protocol, a lightweight, text-based standard for defining procedures and their parameters. This makes it easy to communicate with the API using JSON-formatted requests and receive structured responses.  
+
+### Use Cases  
+- Building integrations for your product or service.  
+- Automating workflows with server-side commands.  
+- Extending your application’s capabilities with remote functions.  
+
+
+## **JSON-RPC Overview**  
+
+### What is JSON-RPC?  
+JSON-RPC is a remote procedure call protocol encoded in JSON. It’s simple, stateless, and transport-agnostic, making it perfect for APIs.  
+
+### Supported Features  
+- **Version:** JSON-RPC 2.0  
+- **Methods:** Flexible method definitions for various tasks.  
+- **Error Handling:** Clear error messages and structured codes.  
+
+:::note[Info]
+
+ To learn more about the Rootstock RPC API, including how to use it, features,  click the button below:
+ 
+<Button href="https://dev.rootstock.io/developers/rpc-api/rootstock/" align="left">Rootstock getBlock Website</Button>
+
+:::
+
+### Key Data Structures  
+- **Request:** 
```

Comment:
Request to get what?

Reply:
If you check this link https://dev.rootstock.io/developers/rpc-api/rootstock/methods/
You will see all of them have same data structure, which is constant
```
{ "jsonrpc": "2.0", "method": "method_name", "params": {}, "id": 1 }
```

Reply:
So if someone wants to send a request via our JSON RPC, thats the structure they will use.
Take a look at these examples
```
curl --location 'https://rpc.testnet.rootstock.io/<api-key>' \
--request POST \
--header 'accept: application/json' \
--header 'Content-Type: application/json' \
--data '{
    "jsonrpc":"2.0",
    "method":"eth_accounts",
    "params":[],
    "id":0
}'
```
And this
```
curl --location 'https://rpc.testnet.rootstock.io/<api-key>' \
--request POST \
--header 'accept: application/json' \
--header 'Content-Type: application/json' \
--data '{
    "jsonrpc":"2.0",
    "method":"eth_blockNumber",
    "params":[],
    "id":0
}'
```

They but have same structure, which is:
```
'{
    "jsonrpc":"2.0",
    "method":"",
    "params":[],
    "id":0
}'
```